packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -138,10 +138,34 @@ export default abstract class BaseCall implements IWebRTCCall {
     this._targetNodeId = what;
   }
 
+  /**
+   * Gets the local stream of the call.
+   * This can be used in a video/audio element to play the local media.
+   * [MediaStream](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream).
+   *
+   * @examples
+   *
+   * ```js
+   * const stream = call.localStream();
+   * document.querySelector('video').srcObject = stream;
+   * ```
+   */
   get localStream() {
     return this.options.localStream;
   }
 
+  /**
+   * Gets the remote stream of the call.
+   * This can be used in a video/audio element to play the local media.
+   * [MediaStream](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream).
+   *
+   * @examples
+   *
+   * ```js
+   * const stream = call.remoteStream();
+   * document.querySelector('video').srcObject = stream;
```

Comment:
```suggestion
   * document.querySelector('audio').srcObject = stream;
```

I think audio is the safest choice here as it should be the most common use case right now